tests/src/test/java/alluxio/client/cli/job/DistributedCommandsStatsTest.java

```diff
@@ -43,6 +44,9 @@
 public class DistributedCommandsStatsTest extends JobShellTest {
   private static final long SLEEP_MS = Constants.SECOND_MS * 15;
   private static final int TEST_TIMEOUT = 45;
+  // When the task is finished, end the waiting. We can check the specific task status later.
+  private static final HashSet<Status> FINISH_STATUS = Sets.newHashSet(Status.CANCELED,
```

Comment:
This is an unstable case. Found from the failed UT log, when the task had failed, it still waited for 45s.
Actually when the task is finished, we can end the waiting. we had checked the specific task status later (the logic already exists).